pcdsdevices/lasers/counters.py

```diff
@@ -0,0 +1,46 @@
+"""
+Module for counters such as frequency counters and time interval counters.
+"""
+
+from ophyd import Component as Cpt
+from ophyd import Device, EpicsSignal, EpicsSignalRO
+
+from pcdsdevices.variety import set_metadata
+
+from pcdsdevices.interface import BaseInterface
+
+
+class Agilent53210A(BaseInterface, Device):
+    """
+    Agililent 53210A frequency counter class.
+
+    For use with IOC ioc/common/agilent5322.
+    """
+    protocol = Cpt(EpicsSignal, ':PROTOCOL_RBV', write_pv=':PROTOCOL',
+                   kind='omitted')
+
+    freq_rbck = Cpt(EpicsSignalRO, ':FREQ_RBCK', kind='normal')
+    freq_rbck_raw = Cpt(EpicsSignalRO, ':FREQ_RBCK_RAW', kind='omitted')
```

Comment:
In the future, I think it'd be nice to have better attribute names - we're not limited to 60 character PV limits in Python after all :)

Reply:
True. Personally I find long attribute names cumbersome, but that shouldn't get in the way of making the attribute name clear and intuitive.